src/frontend/apps/impress/src/features/docs/doc-management/components/items/SimpleDocItem.tsx

```diff
@@ -0,0 +1,62 @@
+import { css } from 'styled-components';
+
+import { Box, Text } from '@/components';
+import { useCunninghamTheme } from '@/cunningham';
+import { Doc } from '@/features/docs';
+import PinnedDocumentIcon from '@/features/docs/doc-management/assets/pinned-document.svg';
+import SimpleFileIcon from '@/features/docs/doc-management/assets/simple-document.svg';
+
+const ItemTextCss = css`
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: initial;
+  display: -webkit-box;
+  line-clamp: 1;
+  -webkit-line-clamp: 1;
+  -webkit-box-orient: vertical;
+`;
+
+type SimpleDocItemProps = {
+  doc: Doc;
+  isPinned?: boolean;
+  subText?: string;
+};
+
+export const SimpleDocItem = ({
+  doc,
+  isPinned = false,
+  subText,
+}: SimpleDocItemProps) => {
+  const { spacingsTokens } = useCunninghamTheme();
+  const spacings = spacingsTokens();
+  return (
+    <Box $direction="row" $gap={spacings.sm}>
+      <Box
+        $direction="row"
+        $align="center"
+        $css={css`
+          background-color: transparent;
+          filter: drop-shadow(0px 2px 2px rgba(0, 0, 0, 0.05));
+        `}
+      >
+        {isPinned ? <PinnedDocumentIcon /> : <SimpleFileIcon />}
+      </Box>
+      <Box>
+        <Text
+          aria-describedby="doc-title"
+          aria-label={doc.title}
+          $size="sm"
+          $variation="1000"
+          $weight="500"
+          $css={ItemTextCss}
+        >
+          {doc.title}
+        </Text>
+        <Text $size="xs" $variation="500" $weight="500" $css={ItemTextCss}>
```

Comment:
In mobile view, it seems to be missing the visibility informations:
![image](https://github.com/user-attachments/assets/5da7db2f-a533-43d6-8a01-e52aee36ad35)